src/App.tsx

```diff
@@ -63,6 +64,16 @@ const Header = ({ activeTab, setActiveTab, setFilters, setResult }) => {
                   >
                     Stars
                   </button>
+                  <button
+                    onClick={() => setActiveTab("fortCalculator")}
+                    className={`${
+                      activeTab === "fortCalculator"
+                        ? activeTabStyles
+                        : inactiveTabStyles
+                    } px-3 py-2 font-medium text-sm rounded-md`}
+                  >
+                    Fort Calculator
```

Comment:
Or maybe Battalion calculator?

Reply:
Not sure it applies, battalions are prefixes. It doesn't have to be a prefix for people to use, maybe another name? Let me know

Reply:
yeah indeed. i dont know i ll let you decide on this